plugins/tiddlywiki/prosemirror/setup/keymap.js

```diff
@@ -19,59 +19,69 @@ var mac = typeof navigator != "undefined" ? /Mac|iP(hone|[oa]d)/.test(navigator.
 function buildKeymap(schema, mapKeys) {
 	var keys = {}, type;
 	function bind(key, cmd) {
-		if (mapKeys) {
+		if(mapKeys) {
 			var mapped = mapKeys[key];
-			if (mapped === false) return;
-			if (mapped) key = mapped;
+			if(mapped === false) return;
+			if(mapped) key = mapped;
 		}
 		keys[key] = cmd;
 	}
 
 	bind("Mod-z", prosemirrorHistory.undo);
 	bind("Shift-Mod-z", prosemirrorHistory.redo);
 	bind("Backspace", prosemirrorInputrules.undoInputRule);
-	if (!mac) bind("Mod-y", prosemirrorHistory.redo);
+	if(!mac) bind("Mod-y", prosemirrorHistory.redo);
 
 	bind("Alt-ArrowUp", prosemirrorCommands.joinUp);
 	bind("Alt-ArrowDown", prosemirrorCommands.joinDown);
 	bind("Mod-BracketLeft", prosemirrorCommands.lift);
 	bind("Escape", prosemirrorCommands.selectParentNode);
 
-	if (type = schema.marks.strong) {
+	type = schema.marks.strong;
+	if(type) {
 		bind("Mod-b", prosemirrorCommands.toggleMark(type));
 		bind("Mod-B", prosemirrorCommands.toggleMark(type));
 	}
-	if (type = schema.marks.em) {
+	type = schema.marks.em;
+	if(type) {
 		bind("Mod-i", prosemirrorCommands.toggleMark(type));
 		bind("Mod-I", prosemirrorCommands.toggleMark(type));
 	}
-	if (type = schema.marks.code)
+	type = schema.marks.code;
+	if(type)
 		bind("Mod-`", prosemirrorCommands.toggleMark(type));
-	if (type = schema.nodes.blockquote)
+	type = schema.nodes.blockquote;
+	if(type)
 		bind("Ctrl->", prosemirrorCommands.wrapIn(type));
-	if (type = schema.nodes.hard_break) {
+	type = schema.nodes.hard_break;
+	if(type) {
 		var br = type, cmd = prosemirrorCommands.chainCommands(prosemirrorCommands.exitCode, function(state, dispatch) {
-			if (dispatch) dispatch(state.tr.replaceSelectionWith(br.create()).scrollIntoView());
+			if(dispatch) dispatch(state.tr.replaceSelectionWith(br.create()).scrollIntoView());
 			return true;
 		});
 		bind("Mod-Enter", cmd);
 		bind("Shift-Enter", cmd);
-		if (mac) bind("Ctrl-Enter", cmd);
+		if(mac) bind("Ctrl-Enter", cmd);
 	}
-	if (type = schema.nodes.list) {
+	type = schema.nodes.list;
+	if(type) {
 		bind("Shift-Tab", prosemirrorFlatList.createDedentListCommand(type));
 		bind("Tab", prosemirrorFlatList.createIndentListCommand(type));
 	}
-	if (type = schema.nodes.paragraph)
+	type = schema.nodes.paragraph;
+	if(type)
```

Comment:
Please use curly braces for all your if's here. The code will be much better to read for humans.